GitUI/UserControls/FileStatusList.cs

```diff
@@ -1070,6 +1070,7 @@ private void FileStatusListLoading()
             FileStatusListView.Groups.Clear();
             FileStatusListView.Items.Clear();
             FileStatusListView.EndUpdate();
+            FileStatusListView.BringToFront();
```

Comment:
Rather set the visibility in dependency on whether the commit is empty (tested with branch mstv/repro/empty_commit).

Reply:
Set visibility is preferred, bring to front disrupts but just changing Visible seem to change order in some situations at least.
I had some problems here...
Whatever that works is OK for me.

Reply:
I will have a look again but the problem is that visibility is already set to false and the label is still visible 🤷

So I tried that as we already play with that to bring the label to front.

Reply:
Can we push the lable to the back rather change the z-order of other controls?